sql/catalyst/src/main/scala/org/apache/spark/sql/catalyst/expressions/stringExpressions.scala

```diff
@@ -53,13 +53,17 @@ import org.apache.spark.unsafe.types.{ByteArray, UTF8String}
  */
 // scalastyle:off line.size.limit
 @ExpressionDescription(
-  usage = "_FUNC_(sep[, str | array(str)]+) - Returns the concatenation of the strings separated by `sep`.",
+  usage = "_FUNC_(sep[, str | array(str)]+) - Returns the concatenation of the strings separated by `sep`, skipping null values.",
```

Comment:
It's a null intolerant expression. Nulls are not skipped and the behavior is expected as null propagation. FYI, https://spark.apache.org/docs/latest/sql-ref-null-semantics.html#null-intolerant-expressions-.

Reply:
It looks like Pg got the wrong answer